test/functionalTest/cases/0945_pagination_with_cprs/pagination_with_cprs_mix_entities.test

```diff
@@ -0,0 +1,842 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Pagination Mixing Local and Remote entitiets
+
+--SHELL-INIT--
+dbInit CB
+dbInit CP1
+brokerStart CB
+brokerStart CP1
+
+--SHELL--
+
+#
+# 01. Create en1, en2/t, en3/t, en4/t in CB (with a small delay to have different creDates)
+# 02. Create en4/ping in CP1
+# 03. Register en4/ping in CB
+# 04. Query all
+# 05. Query offset=0, limit=1
+# 06. Query offset=0, limit=2
+# 07. Query offset=0, limit=3
+# 08. Query offset=0, limit=4
+# 09. Query offset=0, limit=5
+# 10. Query offset=1, limit=1
+#
+
+echo "01. Create en1, en2/t, en3/t, en4/t in CB (with a small delay to have different creDates)"
```

Comment:
For a test script point of view, it is just one step (composed of 4 different NGSI interactions).